instance/management/commands/activity_csv.py

```diff
@@ -123,9 +125,13 @@ def handle(self, *args, **options):  # pylint: disable=too-many-locals
 
                 instance_age = datetime.now(instance.created.tzinfo) - instance.created
 
-                emails = [user.email for user in instance.lms_users.all()]
+                try:
+                    email = instance.betatestapplication_set.get().user.email
+                except BetaTestApplication.DoesNotExist:
+                    email = 'N/A'
```

Comment:
@pomegranited I actually used this as a quick way to determine which Instances were not created via the beta test for this iteration. Double meanings are bad, I know. :)

I think maybe we should hold off on updating this piece until the beta test is over, since I think the only Instances we want to collect this data for right now are the beta testers. Certainly when the beta testing phase it over, we'll want to either change up this command to not rely on BetaTestApplication at all, or remove this command.